CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+
+***NOTE***: Only original source code from you and other people that have
+signed the CLA can be accepted into the repository. This policy does not
+apply to [vendor](vendor/).
+
+### Finding Things That Need Help
+
+If you're new to the project and want to help, but don't know where to start,
+we have a semi-curated list of issues that have should not need deep knowledge
+of the system.  [Have a look and see if anything sounds
+interesting](https://github.com/TykTechnologies/tyk/issues?q=is%3Aopen+is%3Aissue+label%3Ahelp-wanted).
+
+Alternatively, read some of the [many docs on the system](https://tyk.io/tyk-documentation/get-started/), and pick a component that seems
+interesting.  Start with `main()` and read
+until you find something you want to fix.  The best way to learn is to hack!
+There's always code that can be clarified and variables or functions that can
+be renamed or commented.
+
+### `master` is unsafe
+
+We will do our best to keep master in good shape, with tests passing at all times. But in order to move fast, we will make API changes that your application might not be compatible with. We will do our best to communicate these changes and version appropriately so you can lock into a specific version if need be. For stable releases check our tags and `stable` branch, which contains our latest stable release.
+
+
+### Contributing A Patch
+
+If you're working on an existing issue, such as one of the `help-wanted` ones
+above, simply respond to the issue and express interest in working on it.  This
+helps other people know that the issue is active, and hopefully prevents
+duplicated efforts.
+
+If you want to work on a new idea of relatively small scope:
+
+1. Submit an issue describing your proposed change to the repo in question.
+1. The repo owners will respond to your issue promptly.
+1. Clone the repo, develop, and test your changes.
+1. Submit a pull request.
+1. If your proposed change is accepted, and you haven't already done so, sign a
+   Contributor License Agreement (see details above).
+
+If you want to work on a bigger idea, we STRONGLY recommend that you start with
+some bugs or smaller features. It is always better to discuss your idea with our team first, before implementing it.
+
+### Downloading the project
+You need to clone Tyk from GitHub to your GOPATH folder, or alternatively you can run `go get -d github.com/TykTechnologies/tyk` which automatically downloads project to the right path.
+
+### Building the project
+You need to have working Go enironment: see [golang.org](https://golang.org/doc/code.html) for more info on how Go works with code.
+
+To build and test Tyk use built-in `go` commands: `go build` and `go test -v`. If you want to just test a subset of the project, you can pass the `-run` argument with name of the test.
```

Comment:
Well, this section for contributors, and named "Building the project" the project. And it still generate binary, if you want just to test Tyk.

`install` is a tricky command and rarely anyone use it. By default, $GOPATH/bin does not add to the $PATH, so it will only confuse newcomers. And frankly, I did not saw anyone using it in documentation.